vpr/src/base/globals.h

```diff
@@ -8,5 +8,6 @@
 #include "vpr_context.h"
 
 extern VprContext g_vpr_ctx;
+extern const char* _argv0;
```

Comment:
Why is this needed? I don't think this will be accepted to be pushed upstream

Reply:
This is not needed and has been removed.